src/overlays/actors/ovl_Boss_Tw/z_boss_tw.h

```diff
@@ -55,6 +55,37 @@ typedef enum {
     /* 26 */ FWORK_MAX
 } TwFwork;
 
+typedef enum {
+    /*  0 */ TWINROVA_KOTAKE_KOUME_LIMB_NONE,
+    /*  1 */ TWINROVA_KOTAKE_KOUME_LIMB_PELVIS,
+    /*  2 */ TWINROVA_KOTAKE_KOUME_LIMB_LEFT_THIGH,
+    /*  3 */ TWINROVA_KOTAKE_KOUME_LIMB_LEFT_SHIN,
+    /*  4 */ TWINROVA_KOTAKE_KOUME_LIMB_LEFT_FOOT,
```

Comment:
I find the super long "system" name of this enum to be pretty unfortunate. 
perhaps `TWINROVA_SINGLE_` and `TWINROVA_COMBINED_` or something can work
my first instinct was `TWINROVA_WITCH_` or something to identify the split variant, but idk if I like that

Reply:
Could remove `TWINROVA_`?

Reply:
it is a bit weird tho cause twinrova is the larger "actor" system name, which I think should be the same between the two

Reply:
Note the xml does use gKotakeKoume for some stuff

Reply:
One other reason to not remove `TWINROVA` is to help avoid naming collisions with `object_tr` stuff. I'm curious if I should split the enum, since they are technically different skeletons between the two (though I would still need to think of a better name for the assets shared between the two in the xml as Dragorn pointed out).

Reply:
I was going to say they should probably be different, but within override and post limb draw, it is shared between the two. so a seperate enum wouldnt really help anyway

and yea i also think the long names in assets is unfortunate, but I dont have any good suggestions for now and wont hold it up cause of that